crates/types/src/config/bifrost.rs

```diff
@@ -261,6 +262,44 @@ pub struct ReplicatedLogletOptions {
     ///
     /// Value must be between 0 and 1. It will be clamped at `1.0`.
     pub readahead_trigger_ratio: f32,
+
+    /// # Default replication property
+    ///
+    /// Configures the default replication property to be used by the replicated loglet.
+    // Also allow to specify the replication property as non-zero u8 value to make it simpler to
+    // pass it in via an env variable.
+    #[serde_as(
+        as = "serde_with::PickFirst<(serde_with::DisplayFromStr, ReplicationPropertyFromNonZeroU8)>"
+    )]
+    // hide the configuration option from serialization if it is the default
+    #[serde(
+        default = "default_replication_property",
+        skip_serializing_if = "is_default_replication_property"
+    )]
+    // hide the configuration option by excluding it from the Json schema
+    #[cfg_attr(feature = "schemars", schemars(skip, with = "String"))]
+    pub default_replication_property: ReplicationProperty,
+}
+
+/// Helper struct to support deserializing the ReplicationProperty from a [`NonZeroU8`].
+struct ReplicationPropertyFromNonZeroU8;
+
+impl<'de> DeserializeAs<'de, ReplicationProperty> for ReplicationPropertyFromNonZeroU8 {
+    fn deserialize_as<D>(deserializer: D) -> Result<ReplicationProperty, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        NonZeroU8::deserialize(deserializer).map(ReplicationProperty::new)
+    }
+}
+
+fn default_replication_property() -> ReplicationProperty {
+    ReplicationProperty::new(NonZeroU8::new(1).expect("to be non-zero"))
+}
+fn is_default_replication_property(replication_property: &ReplicationProperty) -> bool {
+    replication_property
+        .at_scope_or_greater(LocationScope::Node)
+        .is_some_and(|(scope, value)| *scope == LocationScope::Node && *value == 1)
```

Comment:
I suppose you can use equality check? `replication_property == &ReplicationProperty::new(1)`

Reply:
Good point. Will change it :-)